### PR TITLE
[TACHYON-1428] Documented parts of the tachyon-servers module in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: servers/src/main/java/tachyon/master/file/meta/TtlBucketList.java [hunk 1]
```diff
@@ -35,6 +35,9 @@
    */
   private final ConcurrentSkipListSet<TtlBucket> mBucketList;
 
+  /**
+   * Creates a new list of {@link TtlBucket}s
```
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/file/meta/TtlBucketList.java]: end sentence with '.'

File: servers/src/main/java/tachyon/master/journal/JournalTool.java [hunk 2]
```diff
@@ -52,6 +52,14 @@
   private static boolean sNoTimeout = false;
   private static boolean sHelp = false;
 
+  /**
+   * Reads a journal via
+   * {@code java -cp assembly/target/tachyon-assemblies-0.9.0-SNAPSHOT-jar-with-dependencies.jar \
```
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/journal/JournalTool.java]: It seems wrong to be hardcoding the version into the javadoc.

**Comment by jsimsa** [comment]
@HorizonNet thanks for the PR, left some comments for you

**Comment by HorizonNet** [comment]
@jsimsa Thanks for the comments. I updated the PR accordingly.

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**